src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -368,19 +368,15 @@ private static IEnumerable<SuggestedActionSet> PrioritizeFixGroups(IDictionary<D
                     var fixes = map[diag];
 
                     var priority = fixes.All(s => s is SuppressionSuggestedAction) ? SuggestedActionSetPriority.None : SuggestedActionSetPriority.Medium;
-                    var applicableToSpan = GetApplicableToSpan(diag);
 
-                    sets.Add(new SuggestedActionSet(fixes, priority, applicableToSpan));
+                    // diagnostic from things like build shouldn't reach here since we don't support LB for those diagnostics
+                    Contract.Requires(diag.HasTextSpan);
```

Comment:
This might be true now, but we now support suppressing no-location diagnostics from error list, and when we add LB support for code fixes from error list it might no longer be true - but that presumable would need lot more rewrite of the code in this file.